### PR TITLE
Update/readme april

### PR DESCRIPTION
**Pull request by Gallimathias**
With some delay here the update of the Reame:
* Version has been updated
* Link to azure has been updated
* .vs folder added to gitignore

### CODE REVIEW

File: samples/GetAddressService/Tables/Address.al [hunk 1]
```diff
@@ -3,30 +3,24 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 // ------------------------------------------------------------------------------------------------
 
-table 50100 Address
+table 70051000 Address
```
Comment (StanislawStempin) [inline samples/GetAddressService/Tables/Address.al]: Is this change intended to be part of the pull request?

Comment (Gallimathias) [reply samples/GetAddressService/Tables/Address.al]: No, this change shouldn't really be there :D. Actually, this change should not exist at all. Anyway I fixed it ^^ @StanislawStempin

**Comment by StanislawStempin** [comment]
Thanks!
